tests/pytests/occ_methods.txt

```diff
@@ -0,0 +1,9 @@
+Candidate Structure for other method-tests
+
+_ref_module = {scftype: {ref: {frz: {mp2type: {} for mp2type in ['conv', 'df', 'cd']} for frz in ['true', 'false']} for ref in ['rhf', 'uhf']} for scftype in ['pk', 'df']}
```

Comment:
I didn't convey clearly that what I had in mind was filling in the ref values for mp2.5, mp3, lccd, if you have them handy. If not, ok.

Reply:
I'll probably do this for all 8 `occ` methods at the end. I'm going to want to examine every single `occ` test before I call the project done, whether the method is orbital-optimized or not. Again, `occ` has a way of introducing unexpected bugs.